libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo/network/http/HttpEngine.kt

```diff
@@ -27,18 +25,12 @@ interface HttpEngine : Closeable {
   @Throws(ApolloNetworkException::class, CancellationException::class)
   suspend fun execute(request: HttpRequest): HttpResponse
 
-  @ApolloDeprecatedSince(v4_0_0)
-  @Deprecated("Use close", ReplaceWith("close()"), level = DeprecationLevel.ERROR)
-  fun dispose() {
-  }
```

Comment:
This one is an interesting one because even though is is marked as `ERROR`, nothing prevents users to actually implement the `dispose()` function. All they would see is a warning that the override is not deprecated while the base method is.

Not sure if there is a good solution and I don't really want to keep this forever...

Reply:
Yeah it would be nice if implementing a fun deprecated with `ERROR` level was an error?